tests/lineardev_tests.rs

```diff
@@ -75,7 +67,10 @@ fn validate_sizes(name: &str, block_devs: &Vec<&BlockDev>) -> TestResult<()> {
 
     let dm_dev_size = match get_size(path) {
         Ok(s) => s,
-        Err(_) => panic!("Failed to get size for {} ", name),
+        Err(e) => {
```

Comment:
I'm thinking that the panic! macro is best avoided when there is useful cleanup that could be done.

In this case, if we panic  - the code that calls teardown() doesn't get executed.  So, the user of the test needs to use dmsetup to remove dangling DM devices.